oktapam/resource_security_policy.go

```diff
@@ -787,20 +799,23 @@ func readPrivileges(privilegesAttr []any) ([]*client.SecurityPolicyRulePrivilege
 	return privileges, diags
 }
 
-func readPrivilegeEnabled(privilege any) (bool, bool) {
+func readPrivilegeEnabled(privilege any) (bool, bool, bool) {
```

Comment:
nit Better to just return (map[string] any, hasValue), otherwise require keep on adding new properties.